### DISCUSSION TITLE
Home Assistant OS 10.4

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2650] [Announcements]
## Home Assistant Operating System

* Bump buildroot to 2023.02.3 (#2645) @agners
* Bump buildroot to update dbus-broker (#2633) @agners
* Improve service ordering for udisks2.service (#2632) @agners
* Enable UHID to support Bluetooth input devices (#2623) @ANDuff
* Add kernel configurations for RAUC verity format (#2622) @agners
* Bump buildroot to 2023.02.2 (#2604) @agners

## Build System

* Use hosted GitHub Action runners (#2638) @agners
* Use maintained GitHub Action to upload release assets (#2635) @agners

## Open Virtual Appliance

* Linux: Update kernel 6.1.39 (#2649) @agners
* ova: add support for various ethernet adapters (#2637) @cydia2020

## Generic x86-64

* Linux: Update kernel 6.1.39 (#2649) @agners
* Enable Intel Core/Core2/Atom temperature sensor (#2613) (#2614) @agners
* Enable Realtek 802.11ac wireless chips support (#2600) (#2609) @agners

## Hardkernel ODROID

* Linux: Update kernel 6.1.39 (#2649) @agners
* Add Rockchip RK3568 hardware number generator support (#2578) (#2631) @agners
* Add config options to enable 32-bit ARMv7 instruction support on ODROID-M1 (#2615) @jens-maus

## ASUS Tinker

* Linux: Update kernel 6.1.39 (#2649) @agners

## Khadas VIM Series

* Linux: Update kernel 6.1.39 (#2649) @agners

## Generic aarch64

* Linux: Update kernel 6.1.39 (#2649) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/10.4'>Home Assistant OS 10.4</a>.</em>

**Comment by megaz555** [comment]
Same issues as everyone else is having. Intel NUC, Bluetooth constantly fails since 10.0 release. Tried 10.4, still random disconnects. Going back to 9.5 fixes all the issues!
